operator/v1/types_network.go

```diff
@@ -482,6 +482,20 @@ type GatewayConfig struct {
 	// +kubebuilder:default:=false
 	// +optional
 	RoutingViaHost bool `json:"routingViaHost,omitempty"`
+	// V4InternalMasqueradeSubnet contains the v4 masquerade addresses used internally by
+	// ovn-kubernetes to enable host to service traffic. The host is configured with these
+	// addresses, as well as the shared gateway bridge interface.The values can be changed after
+	// installation.
+	// The defaults are 169.254.169.1 - 169.254.169.4
```

Comment:
We also use the `169.254.169.5` btw for the hairpin service traffic. Since its a subnet, its enough to say 
`Default is 169.254.169.0/16` or whatever is the default "subnet".
Doesn't make sense to quote the IPs within the subnet as defaults right?

Reply:
> Almost there. BTW this can't merge without the OVNK PR right? Probably best to put a hold until then?

right, just wanted this up for reivew

Reply:
@tssurya should we just call it a range then. If the user needs to use those addresses for their infra, are we going to shift the masq addresses to a different subnet or just silde them down in the same subnet?

Reply:
I think it should be a subnet. The defaults right now are:
	V4MasqueradeSubnet              = "169.254.169.0/29"
	V6MasqueradeSubnet              = "fd69::/125"

We can specify that the minimum cidr needs to be at least /29 for v4 and at least /125 for v6

Reply:
@trozet is there a reason that we need to specify that min cidr?